File: Go/p3285.go
package main

import (
	"fmt"
)

func stableMountains(height []int, threshold int) []int {
	stable := []int{}

	for i := 1; i < len(height); i++ {
		if height[i-1] > threshold {
			stable = append(stable, i)
		}
	}

	return stable
}

func main() {
	fmt.Println(stableMountains([]int{1, 2, 3, 4, 5}, 2))
	fmt.Println(stableMountains([]int{10, 1, 10, 1, 10}, 3))
	fmt.Println(stableMountains([]int{10, 1, 10, 1, 10}, 10))
}

File: Rust/p3285.rs
impl Solution {
    pub fn stable_mountains(height: Vec<i32>, threshold: i32) -> Vec<i32> {
        let mut stable: Vec<i32> = vec![];

        for i in 1..height.len() {
            if height[i-1] > threshold {
                stable.push(i.try_into().unwrap());
            }
        }

        stable
    }
}